ionic-test-app/ionic.config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can you rename the directory from `ionic-test-app` to `example` or move this into an `example` folder?